### PR TITLE
Make fields for `AddCommand` optional

### PR DESCRIPTION
**Pull request by McNaBry**

### CODE REVIEW

**Review by tanshiyu1999 — APPROVED** [review]
Looks good to me

**Review by wxwern — CHANGES_REQUESTED** [review]
`Messages#format(Contact)` appears to not be updated to handle the new optional values, which means actions to a contact will display references to `Optional` to the user, such as "`Phone: Optional.empty`". Perhaps those should be changed to something along the lines of "`Phone: None`".

**Review by wxwern — APPROVED** [review]

**Comment by wxwern** [comment]
I've fixed the issue directly, so should be good to go now.

File: src/main/java/seedu/address/logic/Messages.java
Comment (CJ-Lee01) [inline src/main/java/seedu/address/logic/Messages.java:1]: Should we completely exclude this if it is none?

**Review by CJ-Lee01 — APPROVED** [review]
LGTM.

File: src/main/java/seedu/address/logic/Messages.java
Comment (CJ-Lee01) [reply src/main/java/seedu/address/logic/Messages.java:1]: Nvrmind we can have this for now.

**Pull request merged**